### ISSUE TITLE
BSOD with WinFSP and WinmonFS.sys

### ISSUE THREAD

**Issue by Lnd-stoL** [billziss-gh/winfsp#318]
One of the users of our WinFSP-based application encountered a BSOD. The issue is easy to reproduce on users's PC. As far as I can see (I have no experience in driver development) the crash is not in WinFSP driver itself but in WinmonFS.sys (have no idea what is it). 
However WinFSP is in the stack trace so I think WinFSP could potentially be the indirect cause of the crash. 
Anyway I have kernel dumps (attached). 
[071020-39234-01.dmp.zip](https://github.com/billziss-gh/winfsp/files/4904260/071020-39234-01.dmp.zip)
[071020-42421-01.dmp.zip](https://github.com/billziss-gh/winfsp/files/4904261/071020-42421-01.dmp.zip)
[Full dump](https://cloud.mail.ru/public/2VJZ/yQu8nFnK9)

**Comment by billziss-gh** [comment]
I agree that this does not look like that WinFsp is directly responsible. Likely this is an incompatibility between WinFsp and WinmonFS.

However looking online WinmonFS appears to be dangerous malware: https://duckduckgo.com/?q=winmonfs.sys&ia=web

Given this I am not certain there is much we can do about this issue.
